### DISCUSSION TITLE
ExplorerPatcher on Win 11 Pro VM - can't install

### DISCUSSION THREAD

**Discussion by Bmuse21** [valinet/ExplorerPatcher#909] [Q&A] [answered]
Windows 11 Pro ARM - 22000.527
Parallels VM

Hi all,

I'm trying to install ExplorerPatcher on Windows 11 Pro, build 22000.527. This is an ARM 64 VM, but it's an official Microsoft build (not an Insider Preview). I'm able to install and run other Windows apps with no problem.

When I run the EP setup file (whether as Administrator, or not), I get the auth prompt, and click Yes. Then the screen goes white for about 2 seconds, and the desktop returns, with the taskbar buttons/icons sliding up into place from the bottom.

And that's it. I don't see that ExplorerPatcher is installed anywhere, there's nothing in Task Manager, and the taskbar buttons are still grouped into one.

Am I missing something obvious? What should I see, to verify that ExplorerPatcher is installed and working?

Thanks for any insight!
B

**Comment by valinet** [comment, marked as answer]
Reference from https://github.com/valinet/ExplorerPatcher/wiki/Reporting-problems: "**ExplorerPatcher on ARM64** is not supported".

As a note, applications, and especially this kind of application, don't and can't automagically work on a wholly different CPU architecture without someone doing the actual work of porting it.

**Reply by Bmuse21** [reply]
Thank you for the instant reply! And I had indeed missed that. Disappointed that I won't be able to use it... this Win 11 is horrendous. I appreciate the clarification, and your time. 

Best,
B

**Reply by valinet** [reply]
Yeah, the nature of this application means that some things have to be ported over manually. Plus, generally, it requires thoughtful testing. This app hooks into `explorer`'s internals and does things in there in order to achieve certain things. I don't have neither an ARM64 machine, nor any use for one, so developing is kind of impossible; also porting it over is and was of very low interest for me. Maybe someone interested can take up the job, they have my support, as much as that can be. StartAllBack *I think* works on ARM64, if you need a workaround for the Windows 11 UI.

**Comment by tfgmail** [comment]
Hi,

Am also facing this i am trying to deploy the software remotely to our VM x64 based PC ,using pdq deploy but after installation the system goes black i am not able to see anything ,



Someone could please help me with this

**Reply by pyrates999** [reply]
Please start a new discussion.  This one has already been answered.
